internal/config/constants.go

```diff
@@ -47,7 +47,7 @@ const (
 	EnvKESKeyName    = "MINIO_KMS_KES_KEY_NAME"
 	EnvKESClientKey  = "MINIO_KMS_KES_KEY_FILE"
 	EnvKESClientCert = "MINIO_KMS_KES_CERT_FILE"
-	EnvKESServerCA   = "MINIO_KMS_KES_CAPATH"
+	EnvKESServerCA   = "MINIO_KMS_KES_CA_PATH"
```

Comment:
This is already used in k8s we should have a fallback @aead

Reply:
Do we set this in operator? Because, I don't see it used there...

Reply:
So this means this broke MINIO_KMS_KES_CAPATH because k8s uses MINIO_KMS_KES_CA_PATH @aead and we don't have that ENV anymore..